src/sentry/identity/slack/provider.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import absolute_import
+
+from sentry import options
+from sentry.identity.oauth2 import OAuth2Provider
+
+options.register('slack.client-id')
```

Comment:
Set `flags=FLAG_PRIORITIZE_DISK` at least on these since I assume we'll be hardcoding these for us in config.

Reply:
Yeah these will be hardcoded in the configs. [Updated here](https://github.com/getsentry/sentry/pull/6708/files#diff-dc19567bb1802d08f2e32eacb8e0980dR7).